snap/hooks/configure

```diff
@@ -3,4 +3,10 @@
 # Register config options if unset,
 # so users can see available options by running
 # `sudo snap get dcgm`.
-[ -z "$(snapctl get nv-hostengine-port)" ] && snapctl set nv-hostengine-port=
+if [ -z "$(snapctl get nv-hostengine-port)" ]; then
+    snapctl set nv-hostengine-port=""
+fi
+
+if [ -z "$(snapctl get dcgm-exporter-address)" ]; then
+    snapctl set dcgm-exporter-address=""
+fi
```

Comment:
I'm ok with this approach of not be explicit of the values and let the payload to decide, but please add a comment for both to say which is the standard values. This can help future travelers.